openmetadata-ui/src/main/resources/ui/src/components/Metric/RelatedMetrics/RelatedMetrics.tsx

```diff
@@ -154,40 +156,43 @@ const RelatedMetrics: FC<RelatedMetricsProps> = ({
     [onMetricUpdate, relatedMetrics]
   );
 
-  return (
-    <Row gutter={[0, 8]}>
-      <Col span={24}>
-        <Space className="w-full items-center">
-          <Typography.Text
-            className={classNames('right-panel-label', {
-              'font-normal': isUndefined(onMetricUpdate),
-            })}
-            data-testid="header-label">
-            {t('label.related-metric-plural')}
-          </Typography.Text>
-          {!isEdit &&
-            !isEmpty(relatedMetrics) &&
-            permissions.EditAll &&
-            !metricDetails.deleted && (
-              <EditIcon
-                className="cursor-pointer"
-                color={DE_ACTIVE_COLOR}
-                data-testid="edit-related-metrics"
-                style={{ verticalAlign: 'middle' }}
-                width="14px"
-                onClick={() => setIsEdit(true)}
-              />
-            )}
-        </Space>
-      </Col>
+  const header = (
+    <Space className="w-full items-center">
+      <Typography.Text
+        className={classNames({
+          'text-sm font-medium': newLook,
+          'right-panel-label': !newLook,
```

Comment:
if we are revamping then why we want to keep old look?

Reply:
Since this are common entities used across application, not sure if we are able to cover all the pages with new changes. hence keeping an option open for having older UI wherever needed.